_posts/en/newsletters/2021-05-19-newsletter.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Bitcoin Optech Newsletter #149'
+permalink: /en/newsletters/2021/05/19/
+name: 2021-05-19-newsletter
+slug: 2021-05-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter provides updates on the previously proposed
+transaction relay reliability workshop and CVE-2021-31876.  Also
+included are our regular sections describing updates to services and
+client software, new releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Relay reliability workshop scheduled:** as mentioned in [Newsletter
+  #146][news146 workshop], Antoine Riard will be hosting IRC-based
+  meetings to discuss how to make unconfirmed transaction relay more
+  reliable for contract protocols such as LN, coinswaps, and DLCs.  The
+  schedule is,
+
+    - Jun 15th, 19:00--20:30 UTC: guidelines about L2 protocols onchain
+      security design; Coordinated cross-layers security disclosures;
+      full-RBF proposal
+
+    - June 22nd (same time): Generic layer two fee-bumping primitive
+      (such as package relay)
+
+    - June 29th (same time): reserved for additional discussions.
+
+- **CVE-2021-31876 BIP125 implementation discrepancy follow up:**
```

Comment:
Should somewhere in this paragraph link to the discussion thread on the mailing list?